assets/js/_js/projects/read-more-less.js

```diff
@@ -9,18 +9,10 @@
   $(function () {
     let readMoreLess = $(".read-more-less");
     readMoreLess.click(function () {
-      var titleSubtitleUbication = $(this).parent().children();
-      for (var i = 0; i < 3; i++) {
-        document.getElementById("principalPopUp").appendChild(titleSubtitleUbication[i].cloneNode(true));
-      }
-      var description = $(this).parent().parent().parent().find(".markdown-style").children();
-      var descriptionContainer = document.getElementById("descPopUp");
-      while (descriptionContainer.firstChild) {
-        descriptionContainer.removeChild(descriptionContainer.firstChild);
-      }
-      for (var i = 0; i < description.length; i++) {
-        descriptionContainer.appendChild(description[i].cloneNode(true));
-      }
+      var titleSubtitleUbication = $(this).siblings(":header").clone();
```

Comment:
Instead of using the `:header` selector you could add a class to decide what elements should be part of the title.